model/model.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MODEL_H
+#define MODEL_H
+
+struct LumpedMass {
+  float temperature = 0.0;
+  float m = 0.0;
+  float cp = 0.0;
+  LumpedMass(float temperatureInit, float mass, float _cp) {
+    temperature = temperatureInit;
+    m = mass;
+    cp = _cp;
+  };
+  int stepForward(float dt, float heatAddition);
+};
+
+#endif // MODEL_H
```

Comment:
This is the core physics model header.  Once we start down the path of FMU, we'll want to make this struct protected by an `#ifdef __cplusplus`, and build out a C API to interact with the struct.  Or, if we can build the FMU itself in C++, then we're all good.